.github/workflows/build.yml

```diff
@@ -13,7 +13,7 @@ env:
   DOCKER_TOKEN: ${{ secrets.DOCKERHUB_TOKEN }}
   # default prefix for all pushed images,
   # set if you need to push images to another set of repos
-  IMAGE_PRE: ''
+  IMAGE_PRE: 'dev-'
```

Comment:
Will you revert this?

Reply:
Yes, before merging downstream PRs. This requires approval of all related PRs.